### ISSUE TITLE
Add title bar color option

### ISSUE THREAD

**Issue by GKid94** [brave/brave-browser#14389]
Add the option to set the title bar color, and also to set it as the same color as Windows like on Chrome

![Screenshot 2021-02-27 095914](https://user-images.githubusercontent.com/18757054/109382719-91dc2d80-78e2-11eb-8831-470efc6dd0d9.jpg)

**Comment by dj-si** [comment]
I strongly support this change. With every other window I can instantly see if it currently has focus by looking at the title bar, as the colour changes depending on whether it does or doesn't have focus. But Brave's title bar colour never changes, so I cannot easily tell if Brave has focus or not. The additional time it takes (although brief) to confirm if Brave actually has focus is highly annoying. (I'm a power user with decades of experience in the IT industry, so anything that's detrimental to productivity is infuriating.)

Please implement colouring of the Brave title bar to match the Windows system colours for foreground and background windows. Thanks.

**Comment by gregsanto** [comment]
I agree that it's very difficult to tell whether or not the brave window has focus. I've tried several different themes but all of them have the same problem. If Brave were to use the same settings I have specified in Windows 10 then this would not be a problem.

**Comment by fireph** [comment]
I would also really like to see this feature, it makes Brave nearly impossible to tell if it is in focus with the current theme choices.

**Comment by magbe08** [comment]
Make it happen!

**Comment by GKid94** [comment]
Still waiting for this @brave-builds

**Comment by GKid94** [comment]
@kylehickinson @cndouglas @StephenHeaps

**Comment by GKid94** [comment]
Still not working

![Image](https://github.com/user-attachments/assets/33326242-6a26-4bc0-ac9e-b8d6029fb9ae)
![Image](https://github.com/user-attachments/assets/3a6a157e-ce43-41ec-93a4-36ef46b3b7be)
![Image](https://github.com/user-attachments/assets/84630809-af2f-48c2-b71f-8686c4153c8a)

**Comment by remd** [comment]
I would really like to see this changed as well. Isn't this something that Chromium already does? Why was it stripped out?
